data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
+            keywords: ["policy", "legislation"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "GermanZero is creating a climate action plan to meet Germany's 1.5C commitment that will be turned into proposed legislation before 2022. A nation-wide campaign will then try to get that legislation passed in parliament.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Hamburg muss handeln",
+            link: "http://hamburg-muss-handeln.de/",
+            keywords: ["private-sector", "legislation", "local", "grassroots"],
+            label: "hamburg-muss-handeln",
+        },
+        description: {
+            content: "A coalition of 300+ companies and organizations from Hamburg to express support for the demands of Fridays for Future. Also provides advice for upcoming state elections.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Fossil Free",
+            link: "https://gofossilfree.org/",
+            keywords: ["grassroots", "legislation", "private-sector", "network", "ngo"],
+            label: "fossil-free",
+        },
+        description: {
+            content: "A network of local grassroots initiatives that individually petition governments and companies to go fossil free. Local groups are present in virtually all major city and are independent in how they pursue their goals and who they talk to.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2012](https://gofossilfree.org/about/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Scientists for Future",
+            link: "https://www.scientists4future.org/",
+            keywords: ["science", "grassroots", "education", "ngo"],
+            label: "scientists-for-future",
+        },
+        description: {
+            content: "An interdisciplinary group of scientists working for a sustainable future. They support Fridays for Future and established [Lectures for Future](https://www.scientists4future.org/infomaterial/vortragsreihen/) to push the climate crisis onto the agenda in universities.",
```

Comment:
There is more than the lectures for futures. Put it into actions or change the wording.

Reply:
The description should give a broad overview on what an organization is doing but doesn't need to contain all actions. Not sure what you'd consider better here, can you make a suggestion? We have several examples for actions being described in the description, e.g. Ende Gelaende.

Reply:
Ok, agree but I think the broad description should then consist of the main focus of the organization.  For me the focus is to connect scientists, so they can put it into their agenda -> get more lectures on it, pressure public because they are scientists, put up local groups.
My suggestion: ' An interdisciplinary group of scientists who agree on the goals of fridays for future. They try to build up an broad network of scientists by forming [local groups](https://www.scientists4future.org/mitmachen/regionalgruppen/) and holding [conferences](https://www.scientists4future.org/2020/02/s4f-kongress-programmgestaltung/). With their expertise they can pressure public and educate student by putting climate crisis on the agenda of university and more.